### DISCUSSION TITLE
Example Poetry command for build-info for later use with xray scan

### DISCUSSION THREAD

**Discussion by htseal** [jfrog/jfrog-cli#1707] [Q&A] [answered]
I have seen that [jfrog-cli-core](https://github.com/jfrog/jfrog-cli-core/pull/466/files#diff-99ec21ef0ad65d6483ff06a77f94181a2df84589022ea8e759b6abc55de8d3f4) has added support for getting build-info using poetry as the python package and build mananger.

[The documentation](https://www.jfrog.com/confluence/display/CLI/CLI+for+JFrog+Artifactory#CLIforJFrogArtifactory-BuildingPythonPackages) has not been updated and I have not used the cli before so I would like to know now what the expected _jf_ command should be, or if there is specific setup required (as that seems to be the case for some python package managers in the documentation), given that the command will be executed in a directory where the pyproject.toml and poetry.lock file reside?

Should the following command work?
`jf poetry install --build-name=abc --build-number=3`

@talarian1 

The intention is to use this build-info to execute an xray scan of the produced artifacts.

**Comment by talarian1** [comment, marked as answer]
Hi @htseal ,
Once this https://github.com/jfrog/jfrog-cli/pull/1691
will be mrged, the example command you wrote will work. 
After the merge a new version of the Jfrog CLI  will be released and we will update our documentation.

**Reply by htseal** [reply]
Thanks for the reply!
 I see, I thought the code was already merged into release https://github.com/jfrog/jfrog-cli-core/tree/v2.21.2 which is referenced in 
 https://github.com/jfrog/jfrog-cli/blob/adfa5ab675b29729669cc0d0490299d3ae9cf5be/go.mod#L11 corresponding to v2.26.2 of jfrog-cli?
 
 I assume you are right about the 1691 PR.
 I will monitor it :)
